prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -123,4 +137,96 @@ class LivdTableDownload() : CliktCommand(
             }
         return allLinks.filter { it.contains(partialHref) }
     }
+
+    /**
+     * Extracts LIVD table from sheet [sheetName] of the input Excel formate [inputfile] file, converts to
+     * csv format, and output to the CSV format [outpufile] file.
+     * @param sheetName is the LOINC Mapping sheet from the downloaded LOINC data code file.
+     * @param inputfile is the input Excel file name.
+     * @param inputfile is the output CSV file name.
+     * @return true for success and fales for failure.
+     */
+    private fun extractsLivdTable(sheetName: String, inputfile: String, outputfile: String): Boolean {
+
+        try {
```

Comment:
This is a humongous try.  Use tries only on the sections of the code that need it.

Reply:
Done.